NEWS

```diff
@@ -1,3 +1,40 @@
+26-Jan-2017 IGNITION v0.21.0
+
+  Features
+
+    - Add support for networkd drop-ins
+    - Add new binary, ignition-validate, for validating Ignition configs
+    - Add overwrite field to files, directories, and links for deleting
+      preexisting items at the node's path
+    - Add options field to raid for specifying arbitrary mdadm options
+    - Add append field to files for appending to preexisting files
+    - Add support for specifying custom certificate authorities to use when
+      fetching objects over HTTPS
+
+  Changes
+
+    - Validate that partition labels don't contain colons, as sgdisk will
+      silently truncate the label
+    - Remove validate flag from Ignition
+    - Warn when the mode for a file or directory is unset
+    - Make HTTP fetching retry log at info level to make logs show up in console
```

Comment:
`fetching retry log at info level` is unclear.  `Log retries of HTTP fetches at info loglevel so messages appear on console`?